src/pages/listInformation/components/ListInformationResultViewer/ListInformationResultViewer.tsx

```diff
@@ -6,7 +6,7 @@ import { EntityTypeColumn } from '../../../../interfaces';
 
 type ListInformationResultViewerType = {
   userFriendlyQuery?: string,
-  contentVersion?: number,
+  refreshTrigger?: number | boolean,
```

Comment:
What is the meaning when this is a number? Is it still the contentVersion or something else?

Reply:
I just made the interface consistent with the query builder. Additionally, we had an issue with the content version. We can not use it as a refresh trigger because it updates right after the edit list and does not wait when refresh is finished